bucket_D8/libopusenc/specification

```diff
@@ -0,0 +1,35 @@
+DEF[PORTVERSION]=	0.2
+DEF[SOVERSION]=		0.4.2
+DEF[SOMAJOR]=		${SOVERSION:R:R}
```

Comment:
This is wrong, remove this line.  SOMAJOR is defined when SOVERSION is defined (below).